src/policy/feerate.h

```diff
@@ -39,18 +40,21 @@ class CFeeRate
         // We've previously had bugs creep in from silent double->int conversion...
         static_assert(std::is_integral<I>::value, "CFeeRate should be used without floats");
     }
-    /** Constructor for a fee rate in satoshis per kvB (sat/kvB).
+
+    /**
+     * Construct a fee rate from a fee in satoshis and a vsize in vB.
      *
-     *  Passing a num_bytes value of COIN (1e8) returns a fee rate in satoshis per vB (sat/vB),
-     *  e.g. (nFeePaid * 1e8 / 1e3) == (nFeePaid / 1e5),
-     *  where 1e5 is the ratio to convert from BTC/kvB to sat/vB.
```

Comment:
These 3 lines of docs were requested in Nov 2020 review feedback to clarify passing `COIN` to the CFeeRate ctor to obtain a fee rate in sat/vB, like `cc.m_feerate = CFeeRate(AmountFromValue(fee_rate), COIN);`

Since the change in https://github.com/bitcoin/bitcoin/pull/21786/files#diff-63c7710d72381cffe692175dd70e47f1891bf75c3e09249ed826f9df3917d02dL219 we no longer do this, so ACK on removing this documentation now.

Some ideas if you agree and retouch (do the brackets around `[sat/kvB]` have a special meaning in doxygen?)

<details><summary>diff</summary><p>

```diff
@@ -29,7 +29,7 @@ enum class FeeEstimateMode {
 class CFeeRate
 {
 private:
-    /** fee rate in [sat/kvB] (satoshis per 1000 virtualbytes) */
+    /** Fee rate in sat/kvB (satoshis per 1000 virtualbytes) */
     CAmount nSatoshisPerK;
 
 public:
@@ -44,8 +44,8 @@ public:
     /**
      * Construct a fee rate from a fee in satoshis and a vsize in vB.
      *
-     * param@[in]   nFeePaid    The fee paid by a transaction in satoshis
-     * param@[in]   num_bytes   The vsize of a transaction in vbytes
+     * param@[in]   nFeePaid    The fee paid by a transaction, in satoshis
+     * param@[in]   num_bytes   The vsize of a transaction, in vbytes
      */
     CFeeRate(const CAmount& nFeePaid, uint32_t num_bytes);
```
</p></details>

Reply:
square brackets are often used around unit indicators (at least in table headers), i would guess it's an idea like that? 
some amusing discussion: https://hsm.stackexchange.com/questions/5907/old-square-bracket-notation-for-units

i'm fine with dropping them, to be clear

Reply:
Dropped the square brackets.